telegram/replykeyboardmarkup.py

```diff
@@ -16,29 +16,47 @@
 #
 # You should have received a copy of the GNU Lesser Public License
 # along with this program.  If not, see [http://www.gnu.org/licenses/].
-"""This module contains an object that represents a Telegram
-ReplyKeyboardMarkup."""
+"""This module contains an object that represents a Telegram ReplyKeyboardMarkup."""
 
 from telegram import ReplyMarkup, KeyboardButton
 
 
 class ReplyKeyboardMarkup(ReplyMarkup):
-    """This object represents a Telegram ReplyKeyboardMarkup.
+    """
+    This object represents a custom keyboard with reply options.
 
     Attributes:
-        keyboard (List[List[:class:`telegram.KeyboardButton`]]):
-        resize_keyboard (bool):
-        one_time_keyboard (bool):
-        selective (bool):
+        keyboard (List[List[:class:`telegram.KeyboardButton`]]): Array of button rows.
```

Comment:
Not really.. if you don't want any special types, there's really no reason no to use strs